app/models/manager_refresh/inventory_collection.rb

```diff
@@ -1,3 +1,50 @@
+# Usage:
+####################################################################################################################
+# Example 1, storing Vm model data into the DB:
+####################################################################################################################
+#   ################################################################################################################
+#   # Example 1.1 Starting with no vms, lets add vm1 and vm2
```

Comment:
If you're using yarddoc, prefer the `@example` tag here.

Reply:
Meh, I'd go through and just convert these two docs to yarddoc, as a good example of a before and after :)